interface/ceed-basis.c

```diff
@@ -331,11 +331,6 @@ static int CeedBasisApplyAtPointsCheckDims(CeedBasis basis, CeedInt num_elem, co
   if (x_ref != CEED_VECTOR_NONE) CeedCall(CeedVectorGetLength(x_ref, &x_length));
   if (u != CEED_VECTOR_NONE) CeedCall(CeedVectorGetLength(u, &u_length));
 
-  // Check compatibility of topological and geometrical dimensions
```

Comment:
Also needed to allow use of cache variables that are longer than strictly required